### PR TITLE
add star_density property

### PR DESCRIPTION
**Pull request by esheldon**

### CODE REVIEW

**Comment by esheldon** [comment]
additional unit tests coming

File: descwl_shear_sims/simple_sim.py [hunk 1]
```diff
@@ -433,6 +433,13 @@ def __init__(
 
         LOGGER.info('simulating bands: %s', self.bands)
 
+    @property
```
Comment (beckermr) [inline descwl_shear_sims/simple_sim.py:436]: This needs docs in the class level docstring.

**Comment by esheldon** [comment]
I think this might be failing due to python 2.7

**Comment by esheldon** [comment]
no, some change in the miniconda distribution

**Comment by beckermr** [comment]
Yeah. They changed their urls for their installer. Hold on.

**Comment by esheldon** [comment]
works in wget but not curl

**Comment by esheldon** [comment]
I guess wget can get redirected, but curl just silently did nothing

**Pull request merged**